src/main/scala/beam/router/skim/core/AbstractSkimmer.scala

```diff
@@ -65,10 +65,21 @@ abstract class AbstractSkimmerReadOnly extends LazyLogging {
   private[core] var currentIterationInternal: Int = -1
   private[core] var aggregatedFromPastSkimsInternal = Map.empty[AbstractSkimmerKey, AbstractSkimmerInternal]
   private[core] val pastSkimsInternal = mutable.HashMap.empty[Int, Map[AbstractSkimmerKey, AbstractSkimmerInternal]]
-
+  var numberOfRequests: Int = 0
+  var noOfReturnsSkimMapActualValue: Int = 0
```

Comment:
I would call it like numberOfSkimValueFound, or similar

Reply:
Alright, thanks. I'm also not sure about the code written for generation of activities. I hope that is what you were hoping for. When I ran it for the beamville and sf-light scenarios I noticed that only OD skims had requests. The values for other skims were 0.